src/explore-education-statistics-common/src/modules/table-tool/components/PublicationForm.tsx

```diff
@@ -78,142 +115,131 @@ const PublicationForm = ({
-                              />
-                            </DetailsMenu>
-                          ))}
-                        </DetailsMenu>
-                      ))
-                    ) : (
-                      <p>No publications found</p>
+                <p>or</p>
+
+                <div className={styles.optionsContainer}>
+                  <FormRadioGroup
+                    id={`${formId}-themes`}
+                    legend="Select a theme"
+                    legendSize="s"
+                    name="themeId"
+                    small
+                    options={themes.map(theme => {
+                      return {
+                        label: theme.title,
+                        value: theme.id,
+                      };
+                    })}
+                    value={selectedThemeId}
+                    onChange={e => {
+                      setSelectedThemeId(e.target.value);
+                      form.setFieldValue('publicationId', '');
+                      setSearchTerm('');
+                    }}
+                  />
+
+                  <div className={styles.publicationsList} id="publications">
+                    <FormFieldRadioGroup
+                      id={`${formId}-publications`}
+                      legend={
+                        <>
+                          Select a publication
+                          <span
+                            className="govuk-visually-hidden"
+                            aria-live="polite"
+                            aria-atomic="true"
+                          >
+                            {' '}
+                            {`${publications.length} ${
+                              publications.length === 1
+                                ? `publication`
+                                : `publications`
+                            } found`}
+                          </span>
+                        </>
+                      }
+                      legendSize="s"
+                      name="publicationId"
+                      small
+                      options={orderBy(publications, 'title').map(
+                        publication => {
+                          return {
+                            label: searchTerm
+                              ? `${publication.title} (${getThemeForPublication(
+                                  publication.id,
+                                )})`
+                              : publication.title,
+                            value: publication.id,
+                          };
+                        },
+                      )}
+                    />
+
+                    {!publications.length && (
+                      <>
+                        <p>Search or select a theme to view publications</p>
+                        {(searchTerm || selectedThemeId) && (
+                          <p>
+                            <em>No publications found</em>
+                          </p>
```

Comment:
Italics are a little horrible because they aren't really supported by the font. Could use `InsetText` instead:

![image](https://user-images.githubusercontent.com/9917868/173819560-0aacfc28-d0d1-4314-bf9a-279a48f060e2.png)